vaapi/vaapicontext.cpp

```diff
@@ -84,14 +86,21 @@ YamiStatus VaapiConfig::create(const DisplayPtr& display,
     VAConfigID config;
 
     if (!checkProfileCompatible(display, profile)){
-        ERROR("Unsupport profile");
+        std::stringstream msg;
+        msg << "Unsupported profile: " << profile << std::endl;
+        ERROR("%s", msg.str().c_str());
         return YAMI_UNSUPPORTED;
     }
 
     vaStatus = vaCreateConfig(display->getID(), profile, entry, attribList, numAttribs, &config);
 
-    if (!checkVaapiStatus(vaStatus, "vaCreateConfig "))
+    if (!checkVaapiStatus(vaStatus, "vaCreateConfig ")) {
+        std::stringstream msg;
+        msg << "Unable to create config for profile: " << profile
+            << " and entrypoint: " << entry << std::endl;
+        ERROR("%s", msg.str().c_str());
```

Comment:
This is what toString(...) is for... so you don't have to do these extra steps before the logging macros.

Reply:
Since checkVaapiStatus(vaStatus, "vaCreateConfig") already prints "vaCreateConfig: the requested VAEntryPoint is not supported", perhaps just do:

```
ERROR("   VAProfile: %s", toString(profile).c_str());
ERROR("VAEntrypoint: %s", toString(entry).c_str());
```

...to cut down on the error message redundancy.

Reply:
or maybe:

```
ERROR("%s, %s", toString(profile).c_str(), toString(entry).c_str());
```